k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -214,6 +214,14 @@ spec:
               cloudConfig:
                 description: CloudConfiguration defines the cloud provider configuration
                 properties:
+                  awsEBSCSIDriver:
```

Comment:
There are `azure` and `openstack` fields within `cloudConfig`, i wonder if it makes sense to put this in an `aws` field? Given there are other aws-related fields in `cloudConfig` directly so it wouldn't be consistent until we can move them in a future apiVersion.

Reply:
If I remember correctly, we discussed this earlier and opted for keeping consistency. So AWS fields would remain in cloudconfig in this API version.